playground/ssr-noexternal/__tests__/ssr-noexternal.spec.ts

```diff
@@ -1,10 +1,10 @@
 import { expect, test } from 'vitest'
 import { port } from './serve'
-import { page } from '~utils'
+import { isBuild, page } from '~utils'
 
 const url = `http://localhost:${port}`
 
-test('message from require-external-cjs', async () => {
+test.runIf(!isBuild)('message from require-external-cjs', async () => {
   await page.goto(url)
   expect(await page.textContent('.require-external-cjs')).toMatch('foo')
 })
```

Comment:
@sapphi-red I had to skip this test during build. It seems it only works with deps optimization during build.

Reply:
👍 I think it's fine for now. To make this work, we'll need to convert `import foo from 'foo'` into `import module from 'node:module'; const foo = module.createRequire(import.meta.url)('foo')` in this case.